pkg/csi/service/wcp/controller.go

```diff
@@ -1319,6 +1315,9 @@ func (c *controller) ControllerUnpublishVolume(ctx context.Context, req *csi.Con
 							// If VirtualMachine is not found, return success assuming volume is already detached
 							break
 						}
+						if err == cnsvsphere.ErrInvalidVC {
+							return nil, csifault.CSIInternalFault, err
+						}
```

Comment:
Since `cnsvsphere.ErrInvalidVC` is the error that will be returned at last by `ControllerUnpublishVolume` and will be seen when we describe the pod, can we make that error string more clear?